contrib/debian/control

```diff
@@ -14,7 +14,7 @@ X-Python3-Version: >= 3.2
 
 Package: python-subscription-manager
 Architecture: amd64
-Depends: ${python:Depends}, ${misc:Depends}, python-dbus, python-rpm, virt-what, python-debian, python-gobject, python-decorator
+Depends: ${python:Depends}, ${misc:Depends}, python-dbus, python-rpm, virt-what, python-debian, python-gobject, python-decorator, apt-transport-katello
```

Comment:
Better tighten the newly added dependency a bit; since it's an arch:all binary of the same source, please use ` (>= ${source:Version})`

Reply:
Rebased and added your suggestion.